src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -646,56 +646,6 @@ const analyticsEvents = {
       },
     },
   ],
-  'va-pagination': [
```

Comment:
I believe this refers to the web-component, not the react component, meaning it shouldn't be deleted.